tests/contrib/pytest/test_pytest.py

```diff
@@ -698,6 +698,36 @@ def test_will_work():
 
         assert test_span.get_tag(test.FRAMEWORK_VERSION) == pytest.__version__
 
+    def test_pytest_will_report_codeowners(self):
```

Comment:
can we add tests with missing or malformed CODEOWNERS, or with files that do not match any code owners?

Reply:
I have added a test and `try/catch` for line parsing 👍